### DISCUSSION TITLE
Error Updating: Argument 1 passed to App\YetiForce\Updater

### DISCUSSION THREAD

**Discussion by AllOnMyOhm** [YetiForceCompany/YetiForceCRM#14128] [How to] [answered]
Getting the following error on attempting to update:
Argument 1 passed to App\YetiForce\Updater::download() must be of the type array, null given, called in /home/maker/public_html/yeti/modules/Settings/Updates/views/Index.php on line 18

What should I do?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13823 https://github.com/YetiForceCompany/YetiForceCRM/issues/12735 https://github.com/YetiForceCompany/YetiForceCRM/issues/13829

**Reply by AllOnMyOhm** [reply]
Thank you
